modules/bridge/src/main/ts/ephox/bridge/components/menu/Autocomplete.ts

```diff
@@ -0,0 +1,23 @@
+import { FieldSchema, ValueSchema } from '@ephox/boulder';
```

Comment:
This is almost definitely the wrong place for this module, or it needs to be renamed to something other than "autocomplete". All autocompleter modules should be in `components/content`.

Reply:
For the name, we discussed that maybe this should be called `title`?